plugins/api-docs/src/components/GraphQlDefinitionWidget/GraphQlDefinition.tsx

```diff
@@ -44,7 +44,7 @@ type Props = {
   definition: string;
 };
 
-export const GraphQlDefinition = ({ definition }: Props) => {
+export const GraphQlDefinition: React.FC<Props> = ({ definition }: Props) => {
```

Comment:
As a matter of fact we [actively shy away from FC and its ilk](https://backstage.io/docs/architecture-decisions/adrs-adr006), so you can remove that here and in the TRPC widget file

Reply:
Will do. I'm not sure why I added the React.FC on the GraphQLDefinition component here either, must have been a force of habit.